config/routes.rb

```diff
@@ -14,4 +14,6 @@
     resources :practitioners, only: [:index, :show], defaults: { format: "xml" }
   end
 
+  get '/doc', :to => redirect('/doc/index.html')
```

Comment:
Rails uses `/doc/index.html` and finds it in the public folder (at `public/doc/index.html`). Upon review I realized I messed up because the rake task is set to output to `public/docs/` rather than `public/doc/` - this should be fixed. :neutral_face: